lib/rubocop/runner.rb

```diff
@@ -20,6 +20,9 @@ def initialize(path, offenses_by_iteration, loop_start: -1)
         message = 'Infinite loop detected'
         message += " in #{path}" if path
         message += " and caused by #{root_cause}" if root_cause
+        message += ' \n\nPlease update to the latest RuboCop version if not already in use,'
```

Comment:
It _might_ be problematic to have newlines here, especially if other programs need to consume the output. The maintainers can probably say more.

Reply:
@andyw8 

Thanks for the review!
I rewrote it in a form without line breaks.